lib/matplotlib/colors.py

```diff
@@ -1247,25 +1247,27 @@ class BoundaryNorm(Normalize):
     """
     def __init__(self, boundaries, ncolors, clip=False):
         """
-        *boundaries*
-            a monotonically increasing sequence
-        *ncolors*
-            number of colors in the colormap to be used
-
-        If::
-
-            b[i] <= v < b[i+1]
+        Parameters
+        ----------
+        boundaries : sequence
+            Monotonically increasing sequence of boundaries
+        ncolors : int
+            Number of colors in the colormap to be used
+        clip : bool, optional
+            If clip is *True*, out of range values are mapped to *0* if they
```

Comment:
But http://matplotlib.org/devdocs/devel/documenting_mpl.html#formatting says to explicitly not do that! (at least for arguments to functions)